### PR TITLE
Add stats to PointsWriter

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by otoolep** [comment]
@jwilder

**Comment by corylanou** [comment]
Looks good, should still have @jwilder take a look to confirm the stats line up with what we think they do.

File: cluster/points_writer.go [hunk 1]
```diff
@@ -18,6 +19,19 @@ import (
...
+const (
+	statWriteReq            = "req"
+	statPointWriteReq       = "point_req"
+	statPointWriteReqLocal  = "point_req_local"
+	statPointWriteReqRemote = "point_req_remote"
+	statWriteOK             = "write_ok"
+	statWritePartial        = "write_partial"
+	statWriteTimeout        = "write_timeout"
+	statWriteErr            = "write_error"
+	statWritePointHHReq     = "point_hh_req"
```
Comment (jwilder) [inline cluster/points_writer.go]: `point_req_hh` instead?  To match the other 3 patterns.

Comment (otoolep) [inline cluster/points_writer.go]: Sure, sounds good.

**Comment by jwilder** [comment]
One question but :+1: otherwise.

**Pull request merged**